src/Microsoft.Health.Fhir.Core/Features/Operations/DataConvert/DataConvertRequestHandler.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features.Operations.DataConvert.Models;
+using Microsoft.Health.Fhir.Core.Features.Security;
+using Microsoft.Health.Fhir.Core.Features.Security.Authorization;
+using Microsoft.Health.Fhir.Core.Messages.DataConvert;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.DataConvert
+{
+    public class DataConvertRequestHandler : IRequestHandler<DataConvertRequest, DataConvertResponse>
+    {
+        private readonly IFhirAuthorizationService _authorizationService;
+        private readonly IDataConvertEngine _dataConvertEngine;
+        private readonly DataConvertConfiguration _dataConvertConfiguration;
+
+        public DataConvertRequestHandler(
+            IFhirAuthorizationService authorizationService,
+            IDataConvertEngine dataConvertEngine,
+            IOptions<DataConvertConfiguration> dataConvertConfiguration)
+        {
+            EnsureArg.IsNotNull(authorizationService);
```

Comment:
Please pass in the name of the parameter as well.

Reply:
Fixed, thanks